### PR TITLE
Third person ragdoll camera

### PR DESCRIPTION
**Pull request by Fueredoriku**
Ragdoll camera on death:
ragdollCam

Pans over to map circleing after 7 seconds:
ragdollend

### CODE REVIEW

File: Assets/Scripts/RagdollController.cs [hunk 1]
```diff
@@ -53,8 +62,35 @@ public void EnableRagdoll(Vector3 knockbackForce)
         rigidbodyToPush.AddForce(knockbackForce, ForceMode.Impulse);
     }
 
-    public void ReparentCamera(Transform cameraTransform)
+    public void ReparentCamera(Camera camera, Transform input)
```
Comment (toberge) [inline Assets/Scripts/RagdollController.cs:65]: This doesn't reparent the camera anymore, perhaps rename it?

File: Assets/Scripts/RagdollController.cs [hunk 2]
```diff
@@ -53,8 +62,35 @@ public void EnableRagdoll(Vector3 knockbackForce)
...
+    {
+        //camera.transform.parent = cameraParent;
```
Comment (toberge) [inline Assets/Scripts/RagdollController.cs:67]: And this is a live example of an uncommented line :latin_cross:

File: Assets/Scripts/RagdollController.cs [hunk 3]
```diff
@@ -53,8 +62,35 @@ public void EnableRagdoll(Vector3 knockbackForce)
...
+    }
+
+    private void OnDestroy()
     {
-        cameraTransform.parent = cameraParent;
+        if(!camera)
+            return;
+        camera.position = Vector3.zero;
+        camera.localPosition = Vector3.zero;
+        camera.rotation = inputTransform.rotation;
```
Comment (toberge) [inline Assets/Scripts/RagdollController.cs:94]: This doesn't reset the culling mask (not really that necessary I suppose)

**Review by toberge — APPROVED** [review]
looks good to me bruh

File: Assets/Scripts/RagdollController.cs (hunk 3 above)
Comment (Fueredoriku) [reply Assets/Scripts/RagdollController.cs:94]: Ye, it gets automatically changed back on playerManager start

File: Assets/Scripts/RagdollController.cs (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Scripts/RagdollController.cs]: Is fix 👍

File: Assets/Scripts/RagdollController.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/RagdollController.cs]: Will do!

**Pull request merged**